makedebian.pl

```diff
@@ -265,19 +261,15 @@
 print SCRIPT <<EOF;
 #!/bin/sh
 
-# Fix old versions of Webmin that might kill the UI process on upgrade
-if [ -d /etc/webmin ]; then
+# Fix old versions of Webmin that might kill the UI
```

Comment:
I deleted this whole block in a commit just now - please merge it.

Reply:
Done - merged `makedebian.pl` only.

Reply:
I also updated makerpm.pl

Reply:
Yes, I merged that as well.

Reply:
Excellent, thanks